layouts/partials/ecosystem/registry/entry.html

```diff
@@ -46,6 +46,11 @@
                 "installLine" "cargo add %s"
                 "installTemplate" "ecosystem/registry/quickinstall/default.md"
                 "icon" "fab fa-rust")
+            "maven" (dict
+                "urlPattern" "https://central.sonatype.com/%s"
+                "installLine" "mvn install %s"
```

Comment:
`mvn install` does something different (https://maven.apache.org/plugins/maven-install-plugin/), what you will need to show is the snippet that someone needs to add to the `pom.xml` to add the dependency, here

```xml
<dependency>
    <groupId>io.opentelemetry.instrumentation</groupId>
    <artifactId>opentelemetry-okhttp-3.0</artifactId>
    <version>2.8.0-alpha</version>
</dependency>
```

To accomplish that you will need a custom file in `quickinstall`, check the `hex.md` and `collector.md` how that can be done. 

I suggest that for the name of the package you use `io.opentelemetry.instrumentation/opentelemetry-okhttp-3.0`, that way the link will be correct. Note that you will face a challenge then with the XML file, since you need to split that name by the `/` to distribute it across `groupId` and `artifactId`. Hugo has a [split](https://gohugo.io/functions/strings/split/) function for that.